tests/pytests/test_asm.py

```diff
@@ -277,8 +286,33 @@ def import_slot_range_sanity_test(env: Env, query_type: str = 'FT.SEARCH'):
     wait_for_slot_import(shard2, task_id)
     query_shards(env, query, shards, expected, query_type)
 
-def import_slot_range_test(env: Env, query_type: str = 'FT.SEARCH'):
-    n_docs = 2**14
+def parallel_update_worker(env, n_docs, stop_event):
+    """Worker function that continuously updates documents and forces GC"""
+    with env.getClusterConnectionIfNeeded() as con:
+        update_counter = 0
+        while not stop_event.is_set():
+            # Update some unrelated fields that are not part of the query
+            # We'll add a new field 'update_counter' and 'timestamp' that won't affect search results
+            doc_id = random.randint(0, n_docs - 1)
+            key = f'doc-{doc_id}:{{{doc_id % 2**14}}}'
+
+            # Update fields that are not queried in the test
+            con.execute_command('HSET', key,
+                              'update_counter', update_counter,
+                              'timestamp', int(time.time() * 1000),
+                              'extra_data', f'updated_{update_counter}')
+
+            update_counter += 1
+
+            # Force GC collection periodically (every 100 updates)
+            if update_counter % 100 == 0:
+                con.execute_command('DEBUG', 'GC')
```

Comment:
I'm not familiur with this command... I meant to run `forceInvokeGC()` which triggers FT.DEBUG command